python-package/xgboost/sklearn.py

```diff
@@ -132,10 +137,10 @@ class XGBModel(XGBModelBase):
     """
 
     def __init__(self, max_depth=3, learning_rate=0.1, n_estimators=100,
-                 silent=True, objective="reg:linear", booster='gbtree',
-                 n_jobs=1, nthread=None, gamma=0, min_child_weight=1, max_delta_step=0,
-                 subsample=1, colsample_bytree=1, colsample_bylevel=1,
-                 reg_alpha=0, reg_lambda=1, scale_pos_weight=1,
+                 silent=True, objective="reg:linear", booster='gbtree', tree_method='hist',
```

Comment:
Is changing `tree_method` a must?  We do want to change our default tree method in recent future, but preferably through low level `c++` code.

Reply:
Removed `tree_method`.